src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppECSFargate.recipe

```diff
@@ -10,6 +10,13 @@
     "Description": "ASP.NET Core applications built as a container and deployed to Amazon Elastic Container Service (Amazon ECS) with compute power managed by AWS Fargate compute engine. Recommended for applications that can be deployed as a container image. If your project does not contain a Dockerfile, one will be generated for the project.",
     "TargetService": "Amazon Elastic Container Service",
 
+    "DisplayedResources": [
+        {
+            "LogicalId": "FargateServiceLBB353E155",
+            "Description": "Fargate Service"
```

Comment:
"Fargate Service" -> "ECS Service"

Unless this is pointing to the load balancer, then I would call this Application Endpoint like beanstalk and add a separate resource for the ECS service. The Application Endpoint should be first in the list.

Reply:
Fixed